routes/monitor.js

```diff
@@ -12,24 +12,321 @@ router.get('/', ensureAuthenticated, function(req, res) {
     res.render('monitor', { title: 'Monitor', user: req.user });
 });
 
-router.get('/available_runs', ensureAuthenticated, function(req, res){
+// obtain first and last entire in status collection where channels entrie is not empty
+router.get('/get_limits', ensureAuthenticated, function(req, res){
     var db = req.monitor_db;
-    db.listCollections().toArray(function(err, collInfos) {	
-        collInfos = collInfos.sort(function(a, b){
-            return parseInt(a['name'].substr(0, 6)) - parseInt(b['name'].substr(0, 6));
-        });
-	    return res.send(JSON.stringify(collInfos));
+    console.log(db)
+    
+    var collection = db.get("status");
+    var mongo_pipeline = [];
+    
+    // only take reader 0 to 2 and non empty channel entries
+    mongo_pipeline.push({
+        "$match":{
+            "host": {$regex:"reader[0-2]_reader_0", $options:"$i" },
```

Comment:
We will probably be spinning up reader3 soon, so we should tweak the regex to accept it.